ui/js/models/resultsets_store.js

```diff
@@ -490,6 +490,14 @@ treeherder.factory('ThResultSetStore', [
             return 0;
         };
 
+        var getAllUnclassifiedFailureCount = function(repoName) {
```

Comment:
So, this will give you a larger number of unclassified.  The ones regardless of Tiers.  But what we want to get is a smaller number.  One that would show the ``unclassifiedFailureCountForTiers`` - ``any that are filtered out``

Perhaps if you were to make a function kind of like ``updateUnclassifiedFailureCountForTiers`` but one for filters.  You can go through the list of unclassified and call ``thJobFilters.showJob`` on it.  
But, much like for the tiers one, you'd need to recalculate that based on a couple events.  Like if filters changed, or if new jobs were loaded.

There might be a more efficient way of doing that.  This is just a thought.  :)